### PR TITLE
Don't strip Go executables for Windows.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -109,6 +109,18 @@ jobs:
...
+        if: runner.os != 'macOS' && (github.event_name == 'release' || inputs.strip == 'true')
+        run: |
+          strip -s build/host/sdk/bin/*
+
+      - name: Strip tools - Linux
+        # Stripping Go executables on Windows breaks them.
+        if: runner.os == 'Linux' && (github.event_name == 'release' || inputs.strip == 'true')
+        run: |
+          strip -s build/host/sdk/tools/*
+
```
Comment (kasperl) [inline .github/workflows/ci.yml:122]: Drop one newline.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -109,6 +109,18 @@ jobs:
           make BUILD=build-debug BUILD_TYPE=Debug sdk
           ccache -s
 
+      - name: Strip executables - Linux/Windows
+        if: runner.os != 'macOS' && (github.event_name == 'release' || inputs.strip == 'true')
+        run: |
+          strip -s build/host/sdk/bin/*
+
+      - name: Strip tools - Linux
```
Comment (kasperl) [inline .github/workflows/ci.yml:117]: Maybe just drop all the stripping of these on all platforms?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/ci.yml]: done.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml]: done.

**Pull request merged**